packages/riverpod_lint/README.md

```diff
@@ -50,6 +50,7 @@ Riverpod_lint adds various warnings with quick fixes and refactoring options, su
   - [functional\_ref (riverpod\_generator only)](#functional_ref-riverpod_generator-only)
   - [notifier\_extends (riverpod\_generator only)](#notifier_extends-riverpod_generator-only)
   - [avoid\_ref\_inside\_state\_dispose](#avoid_ref_inside_state_dispose)
+  - [missed\_build\_method (riverpod\_generator only)](#missed_build_method-riverpod_generator-only)
```

Comment:
I'd name it notifier_build to match notifier_extends

Reply:
Sounds good.